pygui/__init__.py

```diff
@@ -0,0 +1,391 @@
+from importlib.metadata import version
+import logging
+import queue
+import time
+from typing import Any, Callable, Dict, List, Optional
+
+from observ import reactive, scheduler, watch
+
+from .renderers import DictRenderer, Renderer
+from .types import EffectTag, EventLoopType, Fiber, VNode
+
+
+__all__ = ["create_element", "PyGui", "EventLoopType"]
+__version__ = version("pygui")
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_element(type, props=None, *children) -> VNode:
+    """Create an element description, based on type, props and (optionally) children"""
+    key = props.pop("key") if props and "key" in props else None
+    return VNode(type, reactive(props or {}), children or tuple(), key)
+
+
+class PyGui:
+    def __init__(
+        self,
+        renderer: Renderer = None,
+        *,
+        event_loop_type: EventLoopType = EventLoopType.DEFAULT,
+    ):
+        if renderer is None:
+            renderer = DictRenderer()
+        assert isinstance(renderer, Renderer)
+        self.renderer = renderer
+        self.event_loop_type = event_loop_type
+        if self.event_loop_type is EventLoopType.QT:
+            scheduler.register_qt()  # pragma: no cover
+        elif self.event_loop_type is EventLoopType.DEFAULT:
+            import asyncio
+
+            def request_flush():
+                loop = asyncio.get_event_loop()
```

Comment:
loop = asyncio.get_event_loop_policy().get_event_loop()

Right?

Reply:
Absolutely right, I've updated it throughout.